app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -125,12 +130,26 @@ acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
     let originalTopics = [...topicList.topic_list.topics];
     topicList.topic_list.topics = [topicList.topic_list.topics[0]];
 
+    // visit root first so topic list starts fresh
+    await visit("/");
     await visit("/new");
+    assert.notOk(exists("#dismiss-new-top"));
     await triggerKeyEvent(document, "keypress", "x".charCodeAt(0));
     await triggerKeyEvent(document, "keypress", "r".charCodeAt(0));
     assert.equal(resetNewCalled, 2);
 
     // restore the original topic list
     topicList.topic_list.topics = originalTopics;
   });
+
+  test("click event not fired twice when both dismiss buttons are present", async function (assert) {
```

Comment:
Hmm the fact that this can fire twice if buttons are clicked in a quick succession feels like we're missing a disable on the buttons after the first click.

Reply:
It's not the same button firing twice, it's two separate buttons firing at the same time because both of their selectors are present :)